src/eko/matching_conditions/nnlo.py

```diff
@@ -50,12 +54,29 @@ def A_qq_2_ns(n, sx):
         )
     )
 
-    return constants.CF * constants.TR * a_qq_2
+    a_qq_2_l1 = (
+        2
+        * (
+            -12
+            - 28 * n
+            + 9 * n ** 2
+            + 34 * n ** 3
+            - 3 * n ** 4
+            + 40 * (n * (n + 1)) ** 2 * S1m
```

Comment:
I would collect by S1: this simplifies the prefactor a lot and I think it is numerically good since S1 itself can grow rapidly (and hence shouldn't be yet more enhanced to avoid numbers with big difference in magnitude)

Reply:
more generally collect by S (or any complicated function)

Reply:
I perfectly agree if you are passing by Mathematica in any case, if instead you don't have Mathematica in your stack already think if it's really so expensive: the less we process the expressions the less we have to debug them.

Reply:
Yes, I agree, we can simplify this easily.